### PR TITLE
Fixed print syntax for py3

### PR DESCRIPTION
**Pull request by ingoha**
Includes a note for the venv path; needs to be addressed...

### CODE REVIEW

**Comment by ingoha** [comment]
Amends https://github.com/moinwiki/moin/issues/720

File: contrib/wsgi/moin2.wsgi [hunk 1]
```diff
@@ -26,22 +26,23 @@ moin_dir = '/path/to/moin/dir/where/wikiconfig.py/is/located'
 if sys.platform == 'win32':
     site.addsitedir(moin_dir + '-venv-python/Lib/site-packages')
 else:
+# TODO: the venv is created in ../moin-venv-python3.5 by quickinstall...
```
Comment (ThomasWaldmann) [inline contrib/wsgi/moin2.wsgi]: so, this is a bug because there is "3.5" missing after "python"?

if so, please do a separate commit or a bug report, it doesn't fit into a "fix the prints" commit.

Comment (ingoha) [reply contrib/wsgi/moin2.wsgi]: You are right, sorry - moved this issue to https://github.com/moinwiki/moin/issues/929. The comment will be removed from the PR...

**Comment by ThomasWaldmann** [comment]
Thanks!
